src/app/dashboard/schools/schools.component.html

```diff
@@ -0,0 +1,144 @@
+<div class="grid-container">
+  <h1 class="user__title">Schools</h1>
+  <p class="user__description">all schools available</p>
+
+  <div class="user__table__container">
+    <div class="actions__wrapper">
+      <div class="actions__left">
+        <span>Schools</span>
+        <div class="search__input__wrapper">
+          <app-rounded-input
+            [control]="searchValue"
+            placeholder="Search by name" />
+        </div>
+      </div>
+      <div class="actions__right">
+        <!-- <button
```

Comment:
You still need this?

Reply:
Maybe not. What could we possibly filter by?